### DISCUSSION TITLE
Disable otel_scope_* labels for metrics in prometheus exporter

### DISCUSSION THREAD

**Discussion by boiko-a** [open-telemetry/opentelemetry-dotnet#6004] [Q&A] [answered]
According to the [spec for the Prometheus exporter](https://opentelemetry.io/docs/specs/otel/metrics/sdk_exporters/prometheus/#configuration):
> A Prometheus Exporter MAY support a configuration option to produce metrics without a [scope info](https://opentelemetry.io/docs/specs/otel/compatibility/prometheus_and_openmetrics/#instrumentation-scope-1) metric, or scope labels. The option MAY be named without_scope_info, and MUST be false by default.

I'm curious about scope labels (otel_scope_name, otel_scope_version). Does current implementation provide any way to drop them from output?

**Comment by martincostello** [comment, marked as answer]
As of [1.17.0-beta.1](https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry.Exporter.Prometheus.AspNetCore/CHANGELOG.md#1170-beta1), set the [ScopeInfoEnabled](https://github.com/open-telemetry/opentelemetry-dotnet/tree/main/src/OpenTelemetry.Exporter.Prometheus.AspNetCore#scopeinfoenabled) option to `false`.
